python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -293,41 +293,125 @@ def test_parallel_writes_and_appends_index_validation(arctic_library, finalize_m
 
 @pytest.mark.parametrize("finalize_method", (StagedDataFinalizeMethod.APPEND, StagedDataFinalizeMethod.WRITE))
 class TestFinalizeWithEmptySegments:
-    def test_staged_segment_is_only_empty_dfs(self, lmdb_library, finalize_method):
-        lib = lmdb_library
+    def test_staged_segment_is_only_empty_dfs(self, arctic_library, finalize_method):
```

Comment:
Why?

Reply:
The whole file uses `arctic_library` as a fixture. I added `lmdb_library` by mistake.